### PR TITLE
Scanner first screens

### PR DESCRIPTION
**Pull request by jfquevedol2198**

### CODE REVIEW

File: scanner/.env.sample
Comment (lietu) [inline scanner/.env.sample:1]: The .gitignore showed you the typical name for this, `.env.example`

File: scanner/src/components/Button/index.svelte [hunk 1]
```diff
@@ -24,6 +23,7 @@
     align-items: center;
     justify-content: center;
     gap: 0.5rem;
+    font-weight: bold;
```
Comment (lietu) [inline scanner/src/components/Button/index.svelte:26]: As andrew mentioned elsewhere, it's easier to get the exact right font if you use numbers for weight, e.g. `700` - which you should get from Figma.

File: scanner/src/routes/style.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+* {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+body {
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
```
Comment (lietu) [inline scanner/src/routes/style.css:8]: `overflow: hidden;` is likely unwanted in general and leads to it being broken in some rare cases where our sizes just don't fit on screen

**Comment by meandrewandyou** [comment]
Button now takes full-width, looks like it shouldn't
Screenshot 2023-09-26 165734

File: scanner/src/routes/+layout.svelte [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  <SvelteToast />
+</main>
+
+<style>
+  main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 20px;
```
Comment (meandrewandyou) [inline scanner/src/routes/+layout.svelte:24]: maybe rems?)

File: scanner/src/lib/common.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { toast } from "@zerodevx/svelte-toast"
+
+export function consoleLog(message: string, type: string = "info") {
```
Comment (meandrewandyou) [inline scanner/src/lib/common.ts:3]: I'm not sure if toasts are temporary here. If not - maybe we could rename the function then? It caused some confusions while reviewing the code

Comment (lietu) [reply scanner/src/lib/common.ts:3]: This code is very temporary so it doesn't matter too much.

**Review by lietu — APPROVED** [review]
Looks good!

**Pull request merged**